File: BlazorSplitterComponent/BSJsInterop.cs
using Microsoft.JSInterop;
using System;
using System.Collections.Generic;
using System.Text;
using System.Threading.Tasks;

namespace BlazorSplitterComponent
{
    internal static class BsJsInterop
    {

        internal static Task<bool> Alert(IJSRuntime jsRuntime, string msg)
        {

            return jsRuntime.InvokeAsync<bool>(
                "BsJsFunctions.alertmsg", msg);
        }

        internal static Task<bool> SetPointerCapture(IJSRuntime jsRuntime, string elementID, long pointerID)
        {
            return jsRuntime.InvokeAsync<bool>(
                "BsJsFunctions.setPCapture", elementID, pointerID);
        }
        internal static Task<bool> releasePointerCapture(IJSRuntime jsRuntime, string elementID, long pointerID)
        {
            return jsRuntime.InvokeAsync<bool>(
                "BsJsFunctions.releasePCapture", elementID, pointerID);
        }

    }
}

File: BlazorSplitterComponent/content/BSJsInterop.js
window.BsJsFunctions = {
    alertmsg: function (m) {
        alert(m);
        return true;
    },
    setPCapture: function (el, p) {

        if (document.getElementById(el) !== null) {
            document.getElementById(el).setPointerCapture(p);
            return true;
        }
        else {
            return false;
        }
    },
    releasePCapture: function (el, p) {

        if (document.getElementById(el) !== null) {
            document.getElementById(el).releasePointerCapture(p);
            return true;
        }
        else {
            return false;
        }
    },
};